src/AddressRegistry/StreetName/Events/AddressPostalCodeWasChangedV2.cs

```diff
@@ -21,6 +21,9 @@ public class AddressPostalCodeWasChangedV2 : IStreetNameEvent, IHasAddressPersis
         [EventPropertyDescription("Objectidentificator van het adres.")]
         public int AddressPersistentLocalId { get; }
 
+        [EventPropertyDescription("Objectidentificatoren van de gekoppelde busnummers.")]
+        public List<int> BoxNumberPersistentLocalIds { get; }
```

Comment:
we should prefer IEnumerable<T>, ICollection<T> or IList<T> -- in this case the consumer just needs an IEnumerable<T>, no?

Reply:
Changed.